development/VehicleConfigParams.yaml

```diff
@@ -100,8 +100,8 @@ vehicle_participant_type: "vehicle:car"
 config_speed_limit: 0.0
 
 # Parameters to enable either ROS 1 or ROS 2 (or both) rosbag logging
-use_ros1_rosbag: true
-use_ros2_rosbag: false
+use_ros1_rosbag: false
+use_ros2_rosbag: true
```

Comment:
Lets turn rosbag recording off by default

Reply:
talked offline to leave it on because development folder usually needs debugging